releasenotes/notes/add-grover-primitives-10f81efdba93703d.yaml

```diff
@@ -0,0 +1,21 @@
+---
+features:
+  - |
+
+    :class:`~.Grover` supports the primitives and can use :class:`~.Sampler` to
```

Comment:
Should we adjust this text "and can use :class:`~.Sampler` to" to say something like "and can use a :class:`~.BaseSampler` sampler type instance to". Its not limited to Sampler right, any derived instance of BaseSampler should be good. The example shows the reference "Sampler".